src/arithmetic/arithmetic_expression.c

```diff
@@ -847,6 +859,69 @@ SIValue AR_TRIM(SIValue *argv, int argc) {
 	return trimmed;
 }
 
+SIValue AR_CONTAINS(SIValue *argv, int argc) {
+	assert(argc == 2);
+
+	// No string contains null.
+	if(SIValue_IsNull(argv[0]) || SIValue_IsNull(argv[1])) return SI_BoolVal(false);
+
+	// TODO: remove once we have runtime error handling.
+	assert((SI_TYPE(argv[0]) & SI_STRING) && (SI_TYPE(argv[1]) & SI_STRING));
+
+	const char *hay = argv[0].stringval;
+	const char *needle = argv[1].stringval;
+
+	// See if needle is in hay.
+	bool found = (strstr(hay, needle) != NULL);
```

Comment:
As much as it made me smile... I think we should use more conventional names as ”lookupValue” and ”lookupString” or ”lookupArray”